### DISCUSSION TITLE
Which USRP can be supported by srsRAN?Is there an official list?

### DISCUSSION THREAD

**Discussion by wmwwimor** [srsran/srsRAN_Project#180] [General Help] [answered]
I check the source code .There are some relevant config files:
gnb_rf_b200_tdd_n78_20mhz.yml         mimo_usrp.yml
gnb_rf_b210_fdd_srsUE.yml             mobility.yml
gnb_rf_n310_fdd_n3_20mhz.yml          qam256.yml
So , besides b200  b210  n310 mentioned above，any other types of SDR can be used?

**Comment by asheeshtripathi** [comment]
We have tested on USRP x310 too.

**Reply by wmwwimor** [reply]
Thx for your reply!

**Comment by andrepuschmann** [comment, marked as answer]
Hey - any device supported by UHD and that allows to configure typical NR sample rates for the desired bandwidth, e.g. 30.72Msps for a 20 MHz cell should work just fine. The example files are really just what they are called - examples of configs we've tested and use ourselves.
